libnavigation-core/src/main/java/com/mapbox/navigation/core/OnboardRouterFiles.kt

```diff
@@ -33,14 +33,11 @@ internal class OnboardRouterFiles(
         }
     }
 
-    private fun defaultFilePath(options: OnboardRouterOptions): String {
-        val tilesUri = options.tilesUri
-        val tilesVersion = options.tilesVersion
-        val directoryVersion = "Offline/${tilesUri.host}/$tilesVersion/tiles"
-        return File(applicationContext.filesDir, directoryVersion).absolutePath
-    }
+    private fun defaultFilePath(): String =
+        File(applicationContext.filesDir, DEFAULT_TILES_FOLDER_NAME).absolutePath
```

Comment:
NN doesn't need a complicated path anymore, it's handled on their side 

/cc @mskurydin @etl